stable/gocd/values.yaml

```diff
@@ -127,6 +127,8 @@ server:
       # custom entrypoint scripts that should be run before starting the GoCD server inside the container.
       dockerEntryPoint: scripts
 
+  hostAliases:
```

Comment:
The convention is to document the usage of this property.  See other properties.